frontend/src/api/files.js

```diff
@@ -42,6 +42,16 @@ async function resourceAction(url, method, content) {
   return res;
 }
 
+async function resourceSizeAction(url, method, content) {
+  url = removePrefix(url);
+  let opts = { method };
+  if (content) {
+    opts.body = content;
+  }
+  const res = await fetchURL(`/api/resources/size${url}`, opts);
+  return res;
+}
```

Comment:
I don't think we need 2 separate functions for that. It is always a get request with no body. So resourceSizeAction and foldersize can be merged and simplified